Clean/Table/Basic.lean

```diff
@@ -13,18 +13,20 @@ import Clean.Table.SimpTable
   A row is StructuredElement that contains field elements.
 -/
 @[reducible]
-def Row (F : Type) (S : Type -> Type) [ProvableType S] := S F
+def Row (F : Type) (S : Type → Type) [ProvableType S] := S F
+
+variable {F : Type} {S : Type → Type} [NonEmptyProvableType S]
 
 @[table_norm]
-def Row.get {F : Type} {S : Type -> Type} [struct: ProvableType S] (row : Row F S) (i : Fin struct.size) : F :=
-  let elems := ProvableType.to_elements row
+def Row.get (row : Row F S) (i : Fin (size S)) : F :=
```

Comment:
I made the implicit variables invisible in several places to make the file more readable